tests/director/api/package_api_test.go

```diff
@@ -0,0 +1,243 @@
+	application := registerApplication(t, ctx, appName)
+	defer unregisterApplication(t, application.ID)
+
+	pkgName := "test-package"
+	pkg := createPackage(t, ctx, application.ID, pkgName)
+	defer deletePackage(t, ctx, pkg.ID)
+
+	api := addAPIToPackage(t, ctx, pkg.ID)
+
+	queryApiForPkg := fixAPIDefinitionInPackageRequest(application.ID, pkg.ID, api.ID)
+	app := graphql.ApplicationExt{}
+	err := tc.RunOperation(ctx, queryApiForPkg, &app)
+	require.NoError(t, err)
+
+	actualApi := app.Package.APIDefinition
+	assert.Equal(t, api.ID, actualApi.ID)
+	saveExample(t, queryApiForPkg.Query(), "query api definition")
+
+}
+
+func TestEventDefinitionInPackage(t *testing.T) {
+	ctx := context.Background()
+
+	appName := "app-test-package"
+	application := registerApplication(t, ctx, appName)
+	defer unregisterApplication(t, application.ID)
+
+	pkgName := "test-package"
+	pkg := createPackage(t, ctx, application.ID, pkgName)
+	defer deletePackage(t, ctx, pkg.ID)
+
+	event := addEventToPackage(t, ctx, pkg.ID)
+
+	queryEventForPkg := fixEventDefinitionInPackageRequest(application.ID, pkg.ID, event.ID)
+	app := graphql.ApplicationExt{}
+	err := tc.RunOperation(ctx, queryEventForPkg, &app)
+	require.NoError(t, err)
+
+	actualEvent := app.Package.EventDefinition
+	assert.Equal(t, event.ID, actualEvent.ID)
+	saveExample(t, queryEventForPkg.Query(), "query-event-definition")
+
+}
+
+func TestDocumentInPackage(t *testing.T) {
+	ctx := context.Background()
+
+	appName := "app-test-package"
+	application := registerApplication(t, ctx, appName)
+	defer unregisterApplication(t, application.ID)
+
+	pkgName := "test-package"
+	pkg := createPackage(t, ctx, application.ID, pkgName)
+	defer deletePackage(t, ctx, pkg.ID)
+
+	doc := addDocumentToPackage(t, ctx, pkg.ID)
+
+	queryDocForPkg := fixDocumentInPackageRequest(application.ID, pkg.ID, doc.ID)
+	app := graphql.ApplicationExt{}
+	err := tc.RunOperation(ctx, queryDocForPkg, &app)
+	require.NoError(t, err)
+
+	actualDoc := app.Package.Document
+	assert.Equal(t, doc.ID, actualDoc.ID)
+	saveExample(t, queryDocForPkg.Query(), "query document")
+}
+
+func TestAPIDefinitionsInPackage(t *testing.T) {
+	ctx := context.Background()
+
+	appName := "app-test-package"
+	application := registerApplication(t, ctx, appName)
+	defer unregisterApplication(t, application.ID)
+
+	pkgName := "test-package"
+	pkg := createPackage(t, ctx, application.ID, pkgName)
+	defer deletePackage(t, ctx, pkg.ID)
+
+	inputA := fixAPIDefinitionInputWithName("foo")
+	addAPIToPackageWithInput(t, ctx, pkg.ID, inputA)
+
+	inputB := fixAPIDefinitionInputWithName("bar")
+	addAPIToPackageWithInput(t, ctx, pkg.ID, inputB)
+
+	queryApisForPkg := fixAPIDefinitionsInPackageRequest(application.ID, pkg.ID)
+	app := graphql.ApplicationExt{}
+	err := tc.RunOperation(ctx, queryApisForPkg, &app)
+	require.NoError(t, err)
+
+	apis := app.Package.APIDefinitions
+	require.Equal(t, 2, apis.TotalCount)
+	assertAPI(t, []*graphql.APIDefinitionInput{&inputA, &inputB}, apis.Data)
+
```

Comment:
Could you also save examples of listing APIs/Events/Documents? (and remove the empty lines at the end of these functions :P)